docs/node/package-initializer.md

```diff
@@ -0,0 +1,61 @@
+# @temporalio/create
```

Comment:
Where does this fit into the overall story? Shouldn't this be apart of introduction?

Seems like there is a lot of "set up" for this SDK. Right now, i don't see a clear story here -

Reply:
There's a lot of "set up", yes.
This is page gives more information about this package in the API reference context.
It could probably be restructured to surface some of the important information.